jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -351,39 +349,24 @@ private void attachAction(RootRouter router) {
 	}
```

Comment:
I don't know what is the ActionResource but it seems to be for `Reset Password` and `User Email Validation`.
See https://github.com/slipstream/SlipStreamServer/blob/issue_%23235/jar-service/src/main/java/com/sixsq/slipstream/action/OneShotActionPerformer.java#L41
So this resource should not have any authentication.